app-text/zathura-cb/zathura-cb-0.1.8.ebuild

```diff
@@ -31,6 +31,8 @@ BDEPEND="virtual/pkgconfig"
 
 pkg_postinst() {
 	xdg_desktop_database_update
+
+	optfeature "jpeg support" x11-libs/gdk-pixbuf:=[jpeg]
```

Comment:
`:=` doesn't do anything in here, it's a subslot binder and causes rebuilds on SONAME / ABI updates. Since this doesn't get linked in compile time, or it shouldn't, it doesn't work here either.

Reply:
Should I remove `:=` then?

Reply:
I did before pushing, thought it's faster since I was going to push anyway.